experiments/AMIP/components/atmosphere/climaatmos.jl

```diff
@@ -64,6 +63,22 @@ function atmos_init(::Type{FT}, atmos_config_dict::Dict) where {FT}
     return sim
 end
 
+# Extension of CA.set_surface_albedo! to set the surface albedo to 0.38 at the beginning of the simulation,
+# so the initial callback initialization doesn't lead to NaNs in the radiation model.
+function CA.set_surface_albedo!(Y, p, t, ::CA.CouplerAlbedo)
+    if t == 0
+        FT = CC.Spaces.undertype(axes(Y.c))
```

Comment:
I think we can also use `FT = eltype(Y)`, but either works :)

Reply:
Good point!